src/widget/wlibrarytableview.cpp

```diff
@@ -267,12 +267,15 @@ void WLibraryTableView::focusInEvent(QFocusEvent* event) {
     QTableView::focusInEvent(event);
 
     if (event->reason() == Qt::TabFocusReason ||
-            event->reason() == Qt::BacktabFocusReason) {
+            event->reason() == Qt::BacktabFocusReason ||
+            event->reason() == Qt::OtherFocusReason) {
```

Comment:
I was wondering why Return in the searchbar would jump to the table but the selection trick wasn't working anymore lately (2.4).
I'll pick this for 2.4

Reply:
https://github.com/mixxxdj/mixxx/commit/19a1bdc8639c7bf2dd5d2019bcbe925949e29d55